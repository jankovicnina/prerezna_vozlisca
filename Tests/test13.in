394 592
196 118 3.92
5 108 4.11
121 60 7.34
374 235 9.65
30 334 4.98
209 45 -1.47
109 290 -0.44
371 70 7.15
74 261 8.12
2 266 6.21
377 259 -0.88
27 356 9.35
267 163 6.16
288 286 9.5
124 347 9.4
355 291 0.51
275 50 2.61
85 306 2.99
186 333 4.68
98 8 3.46
120 165 0.21
62 226 7.56
183 149 5.49
108 28 8.7
102 23 7.06
47 339 3.76
16 311 0.36
267 123 9.15
318 201 8.83
269 173 0.72
271 67 -0.57
37 234 -1.26
11 214 3.05
31 338 8.72
9 196 3.31
285 263 1.74
212 86 1.99
369 116 3.94
223 56 0.05
344 342 4.84
185 80 0.61
82 194 2.45
245 148 -0.65
77 97 4.49
32 274 9.68
331 391 3.95
103 388 3.74
375 177 0.62
198 174 8.21
280 104 2.47
281 224 1.88
17 130 8.86
47 58 5.96
363 150 0.36
270 88 9.26
171 386 7.59
300 146 5.74
225 18 7.78
148 157 0.39
67 92 2.48
372 345 2.02
72 119 5.4
196 385 2.36
274 25 1.42
106 218 1.34
351 156 3.12
143 50 3.63
110 30 7.44
7 40 -1.31
236 109 2.8
358 352 3.98
212 138 2.33
237 117 8.57
15 314 5.06
227 358 -0.93
262 77 7.09
306 7 0.11
338 131 9.74
380 337 3.39
76 316 1.98
389 124 0.13
267 291 7.43
56 238 6.67
114 75 0.86
195 283 4.13
73 359 4.01
182 225 0.89
155 106 -1.28
310 326 4.28
288 94 9.18
213 190 9.86
286 120 7.57
368 247 6.64
38 8 1.33
88 171 5.85
203 374 0.16
96 2 2.46
306 25 5.96
159 188 1.94
351 138 9.21
373 200 7.41
19 74 -1.38
161 290 6.73
322 368 1.77
26 127 -0.14
229 220 7.42
352 355 9.35
55 204 5.03
144 43 6.97
340 370 8.03
91 56 3.23
58 366 8.38
69 381 -1.72
233 179 2.71
150 15 8.2
116 124 0.33
297 63 7.93
87 292 2.59
15 169 -0.01
103 93 9.46
14 128 0.61
301 215 2.31
15 182 0.96
258 249 4.53
294 360 -1.29
178 210 6.88
76 197 1.96
28 231 9.47
298 9 5.91
213 207 4.24
22 37 -0.59
335 277 2.56
57 334 4.57
99 199 0.48
83 331 4.07
146 255 3.46
263 312 -0.27
87 114 1.65
50 145 6.75
155 157 -1.07
41 354 4.4
202 208 9.51
210 231 4.33
287 159 9.85
319 83 -1.91
187 108 8.2
183 50 3.77
111 339 5.09
203 222 5.89
320 197 9.15
24 184 9.93
290 5 -1.22
245 210 2.8
377 27 3.68
303 290 0.12
54 316 0.88
328 377 4.45
68 352 1.18
204 230 8.96
191 57 6.73
60 68 -0.43
11 171 7.8
146 78 8.9
362 371 8.88
82 289 8.58
181 262 3.16
157 302 3.12
156 113 9.07
384 175 1.43
221 323 5.3
359 1 0.31
220 69 0.76
313 265 0.1
140 367 9.55
243 364 0.69
328 382 3.8
257 170 9.83
56 309 3.19
197 316 8.45
136 240 -0.77
164 141 4.23
347 117 4.11
329 368 8.68
28 384 0.18
142 92 2.99
386 64 0.88
324 227 8.95
383 104 1.61
207 53 4.35
248 211 3.18
29 73 1.22
34 224 8.64
163 19 2.57
86 369 6.0
129 43 3.87
225 203 -0.02
120 385 5.87
383 21 9.92
4 314 8.91
172 229 3.15
113 111 -1.41
156 347 4.69
174 43 6.57
107 114 6.74
116 285 2.48
65 222 -1.31
125 253 8.48
177 389 3.72
107 12 0.58
70 162 4.52
183 101 7.39
367 79 -0.13
48 341 2.53
251 142 2.04
268 304 0.19
34 264 6.83
325 197 7.24
341 139 -0.98
162 335 2.99
94 163 2.91
284 267 4.45
128 15 4.54
304 280 9.13
234 4 6.5
205 1 7.73
343 393 -1.36
38 98 3.4
242 11 8.31
293 147 0.12
274 154 4.18
281 16 7.63
18 269 5.49
157 174 4.9
219 14 0.12
222 185 7.82
246 387 8.79
299 391 7.18
149 137 9.54
162 254 4.9
359 147 5.56
240 44 -1.14
271 13 2.97
135 242 3.72
324 160 8.83
308 310 0.36
365 172 8.33
287 300 9.02
226 74 4.6
83 10 8.63
202 1 5.38
39 301 3.5
152 353 4.59
238 248 -0.53
115 361 4.52
137 16 6.07
150 335 0.69
68 325 2.2
134 105 5.65
64 245 9.65
109 48 2.27
99 307 -0.63
372 26 3.48
90 212 1.74
243 147 9.36
12 198 4.26
165 320 3.2
250 118 0.32
357 206 2.05
116 346 2.02
220 25 2.99
356 292 8.02
145 62 3.39
253 370 5.79
123 271 3.64
312 343 0.46
309 259 -1.18
201 277 5.73
87 102 7.49
239 254 8.26
20 19 0.91
93 362 8.86
345 305 8.35
315 180 6.76
220 364 -0.24
21 46 9.27
179 217 7.58
259 94 1.49
23 51 8.02
244 14 9.64
141 389 1.54
228 55 -1.34
362 332 -1.72
215 134 3.15
99 392 2.7
190 265 7.94
25 376 6.67
24 310 0.92
320 71 3.55
159 35 5.34
266 20 7.74
75 324 6.57
392 152 4.29
393 107 4.66
66 158 3.21
41 76 6.57
257 266 -0.27
188 210 0.38
143 357 6.8
95 57 4.65
361 345 9.35
9 259 2.09
42 89 8.21
306 221 7.25
128 349 -0.71
322 161 9.08
249 255 8.65
282 336 0.66
130 101 -1.75
79 176 0.89
378 258 8.62
3 34 9.18
317 349 9.96
254 322 0.1
332 257 1.86
259 376 5.04
154 262 6.54
132 155 7.66
302 219 0.18
376 293 6.21
49 340 -0.52
249 108 2.6
54 107 3.93
340 179 4.78
383 24 0.6
356 63 6.99
202 315 5.47
302 147 8.75
240 0 2.23
305 195 8.33
77 392 5.41
348 135 3.62
61 276 0.88
178 209 -0.31
151 361 5.0
36 178 4.23
78 32 1.2
133 68 1.61
119 54 7.33
35 145 3.16
46 309 3.18
354 295 -0.71
350 187 8.01
230 87 2.92
217 3 -0.48
224 97 1.24
20 246 9.32
72 391 1.62
10 183 4.91
387 160 9.38
152 269 6.32
121 220 1.38
60 304 3.82
0 287 7.56
169 143 5.32
184 5 8.53
198 108 1.8
59 121 5.78
188 393 1.19
92 133 7.93
81 318 0.49
88 244 3.53
169 59 -1.95
229 260 4.57
347 82 4.05
131 253 8.73
173 41 1.57
278 61 0.27
1 363 2.98
316 243 2.48
253 294 6.9
144 373 2.48
232 344 -0.22
105 129 1.56
147 241 1.23
370 100 3.33
218 7 3.91
318 101 0.2
226 76 2.22
388 348 7.37
167 223 2.47
254 317 2.73
296 284 -0.58
48 41 8.33
6 319 4.45
379 95 7.41
326 373 1.23
337 299 2.0
43 386 -0.65
150 166 -0.23
119 83 6.54
216 181 9.87
273 244 2.95
116 237 -1.09
195 349 7.69
224 31 6.66
197 6 0.1
340 204 8.47
338 392 1.15
127 268 1.54
360 38 9.73
206 208 0.3
390 44 0.94
242 62 1.53
185 45 7.58
346 47 6.81
208 321 6.24
385 166 6.59
153 18 2.61
279 289 7.59
350 41 4.61
168 308 2.97
43 93 6.13
53 237 1.98
261 213 0.7
236 197 2.95
187 303 1.59
7 182 -0.84
330 122 4.35
291 328 2.33
306 5 5.65
382 273 4.94
175 77 9.78
181 272 0.99
321 85 3.02
391 48 0.14
178 142 5.4
364 378 5.07
157 119 2.96
205 156 -0.22
192 296 9.16
300 269 8.53
255 313 -1.13
200 281 4.18
122 221 6.15
54 251 2.99
158 308 4.52
287 307 2.26
44 288 6.4
180 191 6.56
40 193 2.27
304 386 3.97
309 227 3.95
256 115 3.81
188 50 8.21
201 256 1.29
228 245 4.76
166 12 1.43
392 202 6.96
223 390 5.93
339 179 2.6
276 196 7.95
131 219 5.21
381 151 9.22
48 94 2.57
339 232 0.29
176 392 3.62
300 232 0.19
287 239 3.85
277 351 2.47
63 207 9.94
391 167 -1.62
52 66 -0.43
261 387 5.09
235 72 7.69
339 46 4.65
139 91 6.57
21 282 4.21
191 363 9.77
129 321 4.19
303 228 -0.44
192 317 -1.72
27 296 6.56
295 236 4.43
45 169 7.39
313 286 1.49
194 143 9.79
112 144 2.77
33 327 4.89
47 72 0.69
58 248 -0.17
160 273 3.82
353 32 0.54
259 250 7.76
301 368 -1.09
342 10 6.56
160 379 6.6
290 22 3.4
0 209 3.84
260 371 -0.02
84 279 -0.06
366 60 -0.15
118 65 2.23
309 90 5.45
264 168 8.88
202 225 8.25
43 287 5.8
323 285 -0.95
311 46 0.09
126 96 5.99
231 159 2.36
55 332 3.3
51 233 9.12
204 181 0.51
68 374 -0.45
271 87 5.32
275 252 -0.93
58 228 7.14
141 103 8.02
349 67 3.25
104 185 6.81
211 36 -0.06
193 305 3.39
232 332 4.99
241 42 1.47
144 60 3.16
288 248 7.34
252 148 5.53
97 383 2.35
168 84 9.95
142 205 -1.49
78 24 9.26
131 126 4.45
289 189 5.38
57 231 7.1
313 25 6.13
244 330 -1.84
100 132 5.98
231 184 5.12
224 15 8.03
131 236 7.42
189 86 7.47
287 327 1.59
272 154 0.33
336 29 5.31
214 297 8.12
215 30 -1.22
212 136 8.87
104 270 -0.36
314 188 9.31
173 100 4.84
8 81 8.73
369 354 3.11
247 52 1.45
89 186 8.69
95 298 7.53
151 192 -0.08
60 140 1.88
199 33 9.07
117 39 6.53
326 110 3.67
306 99 4.02
80 164 5.86
345 239 0.25
170 350 4.23
333 153 6.56
57 49 6.21
334 382 -1.92
310 240 6.35
101 380 -0.93
307 300 2.74
246 374 4.59
138 365 2.27
239 216 0.83
88 301 -0.91
376 75 5.61
50 375 0.96
283 296 7.95
299 125 2.86
265 17 0.09
241 62 8.96
344 328 -0.98
76 239 4.86
13 329 2.79
72 229 7.72
292 372 5.62
90 373 -1.2
254 41 7.84
71 275 6.97
174 112 4.01
353 4 3.29
327 278 -1.95
262 202 -1.08
