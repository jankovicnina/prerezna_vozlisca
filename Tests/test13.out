4623 6314 6913 10499 11958 13759 14056 23524 27835 27865
